### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lesterjackson**
Corrections to (a) running build1.psi and (b) formatting in Linux build section

### CODE REVIEW

File: wrappingGsdk/README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ As mentioned, if you're just evaluating the platform or don't have a game server
 
 You can use the zipped file that was created from `build.ps1` script during your creation of a new MPS Build.
 
+> If PowerShell throws up an error message – <b>File cannot be loaded because running scripts is disabled on this system</b>, then you need to enable the `build.ps1` script to run on your Windows 10 computer.  You can enable this by running the folloiwng PowerShell command in advance of running `build.ps1`:
```
Comment (dgkanatsios) [inline wrappingGsdk/README.md:36]: why specifically Windows version "10"? It should be just Windows, right?

Comment (dgkanatsios) [inline wrappingGsdk/README.md:36]: typo: following

File: wrappingGsdk/README.md [hunk 2]
```diff
@@ -33,6 +33,12 @@ As mentioned, if you're just evaluating the platform or don't have a game server
...
+
+```
+Set-ExecutionPolicy Unrestricted
```
Comment (dgkanatsios) [inline wrappingGsdk/README.md:39]: does this need to be run under an admin powershell?

File: wrappingGsdk/README.md (hunk 1 above)
Comment (lesterjackson) [reply wrappingGsdk/README.md:36]: Technically, we can drop the '10' if that's where you're coming from.

File: wrappingGsdk/README.md (hunk 2 above)
Comment (lesterjackson) [reply wrappingGsdk/README.md:39]: Yes, if your script isn't executable, then you must at least make the script executable from an admin Powershell by calling the recommended call above

File: wrappingGsdk/README.md (hunk 1 above)
Comment (dgkanatsios) [reply wrappingGsdk/README.md:36]: yup, thanks!

File: wrappingGsdk/README.md (hunk 2 above)
Comment (dgkanatsios) [reply wrappingGsdk/README.md:39]: gotcha, do we need to specify that in the instructions/readme then?

Comment (lesterjackson) [reply wrappingGsdk/README.md:39]: I'll add it.  Thanks

**Review by lesterjackson — COMMENTED** [review]
Addressed all comments left by Dimitris

**Review by dgkanatsios — APPROVED** [review]

**Pull request merged**